index.html

```diff
@@ -2,15 +2,86 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Responsive Cake webpage</title>
+  <!-- Add a link to your css file here -->
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/9569a39f56.js" crossorigin="anonymous"></script>
+
 </head>
 
 <body>
-    <!-- Add your markup here -->
+  <!-- Add your markup here -->
+  <header id="header">
+<section class="header">
+    <img src="./img/cakelogo.png" alt="K-logo" class="logo" width="70px" />
+    <button class="hamburger" id="hamburger">
+      <i class="fas fa-bars"></i>
+    </button>
+    <p class="intro">The best cakes in town delivered to your door</P>
+</section>
+  </header>
+  <main>
+  <nav id="nav-bar">
+
+    <ul class="nav-links">
+      <li>
+        <a href="#home">HOME</a>
+      </li>
+      <li><a href="#cakes">CAKES</a></li>
+      <li><a href="orderin">ORDERIN</a></li>
+      <li><a href="#lessons">LESSONS</a></li>
+      <li><a href="#about">ABOUT</a></li>
+    </ul>
+  </nav>
+
+
+  
+    <section id="home" class="home">
+      <h1 class="welcome">Welcome</h1>
```

Comment:
This and a lot of other items could use some more styling. Keep in mind you don't have to create a class for everything. For example, If you know all h1's (aka main titles) should be a larger font with extra padding, then apply that to the h1 tag and not a class. Will make coding a lot quicker as you won't have to remember what classes to what.   Dont stress, that design will come with time .

Reply:
The class "welcome" is not for styling 'h1' I used it to place the 'welcome area' on the position I want it on the grid.